cmake/Scripts/GenerateSleefHeader.cmake

```diff
@@ -19,23 +10,31 @@ if(OPTION_SHOW_CONFIG)
 endif(OPTION_SHOW_CONFIG)
 
 # Generate header for x86
+  set(run1 2 4 __m128d __m128 __m128i __m128i __SSE2__)
```

Comment:
I mean, you do not need to refer to a specific set of parameters anywhere. You either call all the runs in sequence or you don't. 
There is no context in which you need to which set is run6 and which set is run9.